framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/AbpIoSourceCodeStore.cs

```diff
@@ -51,7 +51,8 @@ public class AbpIoSourceCodeStore : ISourceCodeStore, ITransientDependency
             var latestVersion = await GetLatestSourceCodeVersionAsync(name, type);
             if (version == null)
             {
-                version = latestVersion;
+                version = latestVersion ?? throw new CliUsageException(
+                              "The remote service is currently unavailable, please specify the version (like abp new Acme.BookStore -v 2.3.0(Make sure you have a template cache locally))!");
```

Comment:
Maybe we can output the cached templates version? Or open the abp templates cache folder.

![image](https://user-images.githubusercontent.com/6908465/77498622-09137100-6e8b-11ea-80b5-a284012a9470.png)